### PR TITLE
*: install gsed, gfind, gxargs, gpatch for better BSD compatibility

### PR DESCRIPTION
**Pull request by mgorny**
As a continuation of the previous effort, after covering `gcpio` and `gtar`, let's add other tools that the BSD profiles mentioned: `gsed`, `gfind`, `gxargs`, `gpatch`. As previously, the GNU versions are installed as `g*` and symlinked to their base name to make it easy to relink them to alternative implementations.

### CODE REVIEW

File: sys-apps/findutils/findutils-4.9.0-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	# We don't build locate, but the docs want a file in there.
+	emake -C locate dblocation.texi
+	default
+}
+
+src_install() {
+	default
+
+	# rename to gfind, gxargs for better BSD compatibility
+	mv "${ED}"/usr/bin/{,g}find || die
```
Comment (thesamesam) [inline sys-apps/findutils/findutils-4.9.0-r2.ebuild:91]: Can we use --program-prefix in configure instead to avoid needing the mvs?

**Comment by SoapGentoo** [comment]
also, EAPI 8 the lot? We can use EAPI 8 in base now.

**Comment by mgorny** [comment]
Done and done.

**Review by thesamesam — APPROVED** [review]

**Comment by mgorny** [comment]
Included switching cpio and tar to `--program-prefix`. For the latter, it's basically "choose your poison". Either we `mv` tar and rmt, or we `mv` back the other weird tools. I'm wondering if anyone really pays attention to that stuff, though. `USE=-minimal` was weird, to say lightly. `backup` is renamed to `backup-tar` but `backup.sh` is not.

**Pull request closed without merging**